### ISSUE TITLE
[feature] ability to add source tarball as a file option

### ISSUE THREAD

**Issue by sctskw** [panitw/easy-rpm#50]
NPM Pack is a nice feature of npm that creates a tarball of your package, minus unnecessary dependencies and a few other things. It would be great to be able to provide this source tarball in the files option like so:

```
files: ['path/to/source.tgz']
```

This should include all the proper directories that you'd want using the tarball as a source instead of having to manually redefine them in the Gruntfile.

**Comment by paulcoyle** [comment]
Just so I understand, this would involve unpacking the gzipped tarball and using the contents for building the RPM?

**Comment by sctskw** [comment]
That's correct. Although, now that I think about it, it probably wouldn't make sense to list this under "files", as once you list a tarball, I wouldn't think you would want to join it with anything else.

**Comment by paulcoyle** [comment]
That's similar to what I was thinking.  I'm unsure if it should be an either-or usage (where only a files section or a tarball section can be used) or if it is worth trying to negotiate including the tarball's contents into the files (order would obviously be a factor here).  I'm leaning towards the former because of the complexity of the later.
